frontend/src/components/dashboard/aliases/AddressPickerModal.module.scss

```diff
@@ -94,15 +94,14 @@
         cursor: pointer;
         border-radius: $border-radius-sm;
         color: $color-blue-50;
-        padding: $spacing-sm $spacing-2xl;
+        padding: $spacing-sm $spacing-md;
 
         &:hover {
-          background-color: $color-blue-50;
-          color: $color-white;
+          color: $color-link-hover;
```

Comment:
Oh nice, I didn't know about this var. That said, I don't think this hover state is accessible though, since the colour differs so little from the original colour (at first I even thought there was no hover state at all), in addition to colour being the only modality. Which I guess we're doing elsewhere too, but maybe something to bring up with Eduardo.

(Likewise for the "cancel" buttons in `AliasDeletionButton.module.scss` and `ConfirmationForm.module.scss` I suppose.)

Reply:
Yeah - i'll bring this hover state up with Eduardo. Just following the Nebula docs for now.